connectivity_summary.py

```diff
@@ -46,7 +52,10 @@ def arg_to_date(arg):
     print("No experiments loaded; bailing out.")
     sys.exit(-1)
 
-expts = all_expts.select(start=args.start, stop=args.stop, region=args.region)
+for i, ex in enumerate(all_expts._expts):
```

Comment:
No, `ex` is just a reference to each experiment within `all_expts`, so you should still do `all_expts.select(...)`